specification/powerbiprivatelinks/resource-manager/Microsoft.PowerBI/stable/2020-06-01/powerbiprivatelinks.json

```diff
@@ -0,0 +1,1627 @@
+      "get": {
+        "tags": [
+          "PowerBIResources"
+        ],
+        "operationId": "PowerBIResources_ListByResourceName",
+        "x-ms-examples": {
+          "List private link resources in a Azure resource": {
+            "$ref": "./examples/PowerBIResources_ListByResourceName.json"
+          }
+        },
+        "description": "Gets all the private link resources for the given Azure resource.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AzureResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/TenantResource"
+              }
+            },
+            "headers": {}
+          },
+          "default": {
+            "description": "Default Response. It will be deserialized as per the Error definition specified in the schema. Exception will be thrown.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "PowerBIResources"
+        ],
+        "operationId": "PowerBIResources_Create",
+        "x-ms-examples": {
+          "Creates or updates private link service resource": {
+            "$ref": "./examples/PowerBIResources_Create.json"
+          }
+        },
+        "description": "Creates or updates a Private Link Service Resource for Power BI.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AzureResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ClientTenantIdHeaderParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "description": "Tenant resource to be created or updated.",
+            "schema": {
+              "$ref": "#/definitions/TenantResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "$ref": "#/definitions/TenantResource"
+            },
+            "headers": {}
+          },
+          "default": {
+            "description": "Default Response. It will be deserialized as per the Error definition specified in the schema. Exception will be thrown.",
```

Comment:
>Default Response. It will be deserialized as per the Error definition specified in the schema. Exception will be thrown. [](http://example.com/codeflow?start=28&length=120)

Consider better wording.

Reply:
updated